examples/PlotPointwiseGFRE.ipynb

```diff
@@ -0,0 +1,212 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We compare 3-body features with their mapping to the reproducing kernel Hilbert space (RKHS) projected to the sample space using the nonlinear radial basis function (RBF) kernel\n",
+    "\n",
+    "$$ k^{\\textrm{RBF}}(\\mathbf{x},\\mathbf{x}') = \\exp(-\\gamma \\|\\mathbf{x}-\\mathbf{x}'\\|^2),\\quad \\gamma\\in\\mathbb{R}_+ $$\n",
+    "\n",
+    "The projected RKHS features are computed using the eigendecomposition of the positive-definite kernel matrix $K$\n",
+    "\n",
+    "$$ K = ADA^T = AD^{\\frac12}(AD^{\\frac12})^T = \\Phi\\Phi^T$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def compute_standardized_rbf_rkhs_features(features, gamma):\n",
+    "    # standardize features\n",
+    "    features = StandardFlexibleScaler().fit_transform(features)\n",
+    "    \n",
+    "    # compute \\|x - x\\|^2\n",
+    "    squared_distance = ( np.sum(features ** 2, axis=1)[:, np.newaxis]\n",
+    "                                  + np.sum(features ** 2, axis=1)[np.newaxis, :] - 2 * features.dot(features.T) )\n",
+    "    \n",
+    "    # computer rbf kernel\n",
+    "    kernel = np.exp(-gamma * squared_distance)\n",
+    "    \n",
+    "    # center kernel\n",
+    "    kernel = KernelCenterer().fit_transform(kernel)\n",
+    "    \n",
+    "    # compute D and A\n",
+    "    D, A = np.linalg.eigh(kernel)\n",
+    "    \n",
+    "    # retain features associated with an eigenvalue above 1e-9 for denoising\n",
+    "    select_idx = np.where(D > 1e-9)[0]\n",
+    "    \n",
+    "    # compute rkhs features\n",
+    "    rbf_rkhs_features = A[:, select_idx] @ np.diag(np.sqrt(D[select_idx]))\n",
+    "    \n",
+    "    # standardize rkhs features,\n",
+    "    # this step could be ommitted since it is done my the reconstruction measure by default\n",
+    "    standardized_rbf_rkhs_features = StandardFlexibleScaler().fit_transform(rbf_rkhs_features)\n",
+    "    return standardized_rbf_rkhs_features\n",
+    "    \n",
+    "\n",
+    "gamma = 1\n",
+    "rbf_power_spectrum_features = compute_standardized_rbf_rkhs_features(power_spectrum_features, gamma=gamma)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# some split into train and test idx\n",
+    "idx = np.arange(len(power_spectrum_features))\n",
+    "np.random.seed(42)\n",
+    "np.random.shuffle(idx)\n",
+    "train_idx = idx[0::2]\n",
+    "test_idx = idx[1::2]\n",
+    "\n",
+    "print(\"Computing pointwise GFRE...\")\n",
+    "# pointwise global reconstruction error of bispectrum features using power spectrum features\n",
+    "power_spectrum_to_bispectrum_pointwise_gfre = pointwise_global_reconstruction_error(\n",
+    "            power_spectrum_features, bispectrum_features, train_idx=train_idx, test_idx=test_idx)\n",
```

Comment:
Same, I would split this a bit for readability. Something like

```py
power_spectrum_to_bispectrum_pointwise_gfre = pointwise_global_reconstruction_error(
    power_spectrum_features, 
    bispectrum_features, 
    train_idx=train_idx,
    test_idx=test_idx
)
```

And similar below